common/metrics/defs.go

```diff
@@ -267,8 +267,8 @@ const (
 	PersistenceGetClosedWorkflowExecutionScope
 	// PersistenceVisibilityDeleteWorkflowExecutionScope is the metrics scope for persistence.VisibilityManager.DeleteWorkflowExecution
 	PersistenceVisibilityDeleteWorkflowExecutionScope
-	// PersistenceVisibilityDeleteUninitializedWorkflowExecutionScope tracks DeleteUninitializedWorkflowExecution calls made by service to persistence layer
-	PersistenceVisibilityDeleteUninitializedWorkflowExecutionScope
+	// PersistenceDeleteUninitializedWorkflowExecutionScope tracks DeleteUninitializedWorkflowExecution calls made by service to persistence layer
+	PersistenceDeleteUninitializedWorkflowExecutionScope
```

Comment:
I think you'll need to keep the word `visibility` on this one? This is applying to different table then the normal executions table. It's for when CAAS us being used to list workflows instead of elasticsearch (we have visibility table for this)

Reply:
It is used only inside VisibiltyManager, so Visibility prefix is not required. I'm thinking of separating this metrics by manager so it would be easier to track which component emits this, but for now it should be fine.